### PR TITLE
Fixes for xcode 10.2.1

### PR DESCRIPTION
**Pull request by nrostanml**
## Description
Fixes issues on xcode 10.2.1
## Checks
Are you sure you followed all those steps? If so, repeat after me "I solemnly swear that ...":
   - [ ] I have coded an example inside the Demo App,
   - [ ] I have added proper tests,
   - [x] I have modified the Changelog.md file,
   - [ ] I have updated GitHub wiki,
   - [ ] I have the explicit approval of one or more members of the UX Team,
   - [ ] I have checked that this code is ObjC compliant.
## Change type
This is easy, let us know what this code is about:
   - [ ] This code adds a new component
   - [x] This code includes improvements to an existent component
   - [ ] This code improves or modifies ONLY the Demo App.

### CODE REVIEW

**Comment by furyGorgoryAdmin** [comment]
### Informative Menu <h2>  ### 

### Use any of these tags in a pull request description or leave a comment ### 
| Tag | Description |
|---|---|
| **[canary]**| Creates a build that can only be deployed with canary strategy. | 
| **[build]** |  Creates version or build from a branch that has been tagged as "no releasable". | 
| **[name:version-name]** |Specify the name of the build. For example: [build] [name:1.2.3-name] or [canary] [name:1.2.3-name].|
| **[major]** | To automatically increase the first position of the version. For example: [build] [major] or [canary] [major].|
| **[minor]** | To automatically increase the second position of the version. For example: [build] [minor] or [canary] [minor]. |
| **[patch]** | To automatically increase the third position of the version. For example: [build] [patch] or [canary] [patch].|  

[For more information about ci-cd](http://docs.furycloud.io/?lang=esp&section=businessassurance)
[For more information about semver](https://semver.org/#summary)

File: Example/AndesUI/Messages/Views/MessageViewController.swift [hunk 1]
```diff
@@ -120,7 +120,7 @@ extension MessageViewController: UIPickerViewDataSource {
     }
 
     func numberOfComponents(in pickerView: UIPickerView) -> Int {
-        1
+        return 1
```
Comment (nrostanml) [inline Example/AndesUI/Messages/Views/MessageViewController.swift]: implicit return es una feature de swift 5.1

Comment (nsuarezml) [reply Example/AndesUI/Messages/Views/MessageViewController.swift]: Un feature polémico!

File: LibraryComponents/Classes/AndesMessage/Config/AndesMessageViewConfig.swift [hunk 2]
```diff
@@ -29,4 +29,22 @@ internal struct AndesMessageViewConfig {
 
     var primaryActionText: String?
     var secondaryActionText: String?
+
+    init(backgroundColor: UIColor, pipeColor: UIColor, textColor: UIColor, titleText: String?, bodyText: String, icon: UIImage?, iconBackgroundColor: UIColor, isDismissable: Bool, dismissIconColor: UIColor) {
```
Comment (nrostanml) [inline LibraryComponents/Classes/AndesMessage/Config/AndesMessageViewConfig.swift]: se usaba el init implicito con parametros que ya tenian valores por default como opcionales, esto es soportado en swift 5.1 y no compila en xcode 10, por lo tanto tuve que agregar este init

**Pull request merged**